<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindowClass</class>
 <widget class="QMainWindow" name="MainWindowClass">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>662</width>
    <height>596</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>CETC58 ADC Analyzer</string>
  </property>
  <property name="dockNestingEnabled">
   <bool>false</bool>
  </property>
  <property name="dockOptions">
   <set>QMainWindow::AllowTabbedDocks|QMainWindow::AnimatedDocks</set>
  </property>
  <widget class="QWidget" name="centralWidget">
   <property name="sizePolicy">
    <sizepolicy hsizetype="Minimum" vsizetype="Minimum">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <property name="maximumSize">
    <size>
     <width>1</width>
     <height>1</height>
    </size>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout">
    <property name="sizeConstraint">
     <enum>QLayout::SetDefaultConstraint</enum>
    </property>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>662</width>
     <height>21</height>
    </rect>
   </property>
   <widget class="QMenu" name="menu">
    <property name="title">
     <string>文件(&amp;F)</string>
    </property>
    <addaction name="actionLoadData"/>
    <addaction name="actionSaveData"/>
    <addaction name="separator"/>
    <addaction name="action_3"/>
   </widget>
   <widget class="QMenu" name="menuSettings">
    <property name="title">
     <string>设置(&amp;S)</string>
    </property>
    <addaction name="actionSpiCtrl"/>
    <addaction name="actionFftDepth"/>
   </widget>
   <widget class="QMenu" name="menu_T">
    <property name="title">
     <string>测试类型(&amp;T)</string>
    </property>
    <addaction name="actionSpectrum"/>
    <addaction name="actionLogic"/>
   </widget>
   <widget class="QMenu" name="menuWindow">
    <property name="title">
     <string>窗口(&amp;W)</string>
    </property>
   </widget>
   <widget class="QMenu" name="menuHelp">
    <property name="title">
     <string>帮助(&amp;H)</string>
    </property>
    <addaction name="action_7"/>
    <addaction name="action_AboutAdcAnalyzer"/>
   </widget>
   <addaction name="menu"/>
   <addaction name="menu_T"/>
   <addaction name="menuSettings"/>
   <addaction name="menuWindow"/>
   <addaction name="menuHelp"/>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <widget class="QDockWidget" name="dockWidgetWave">
   <property name="sizePolicy">
    <sizepolicy hsizetype="Expanding" vsizetype="Preferred">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <property name="allowedAreas">
    <set>Qt::RightDockWidgetArea</set>
   </property>
   <property name="windowTitle">
    <string>Time-domain</string>
   </property>
   <attribute name="dockWidgetArea">
    <number>2</number>
   </attribute>
   <widget class="QWidget" name="dockWidgetContents_4"/>
  </widget>
  <widget class="QDockWidget" name="dockWidgetFFT">
   <property name="sizePolicy">
    <sizepolicy hsizetype="Expanding" vsizetype="Preferred">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <property name="allowedAreas">
    <set>Qt::RightDockWidgetArea</set>
   </property>
   <property name="windowTitle">
    <string>Freq-domain</string>
   </property>
   <attribute name="dockWidgetArea">
    <number>2</number>
   </attribute>
   <widget class="QWidget" name="dockWidgetContents_5"/>
  </widget>
  <widget class="QDockWidget" name="dockWidgetLogicWave">
   <property name="sizePolicy">
    <sizepolicy hsizetype="Expanding" vsizetype="Preferred">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <property name="allowedAreas">
    <set>Qt::RightDockWidgetArea</set>
   </property>
   <property name="windowTitle">
    <string>Logic Wave</string>
   </property>
   <attribute name="dockWidgetArea">
    <number>2</number>
   </attribute>
   <widget class="QWidget" name="dockWidgetContents_2"/>
  </widget>
  <widget class="QDockWidget" name="dockWidgetControlPanel">
   <property name="sizePolicy">
    <sizepolicy hsizetype="Minimum" vsizetype="Preferred">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <property name="maximumSize">
    <size>
     <width>240</width>
     <height>524287</height>
    </size>
   </property>
   <property name="features">
    <set>QDockWidget::DockWidgetClosable</set>
   </property>
   <property name="allowedAreas">
    <set>Qt::LeftDockWidgetArea</set>
   </property>
   <property name="windowTitle">
    <string>Control Panel</string>
   </property>
   <attribute name="dockWidgetArea">
    <number>1</number>
   </attribute>
   <widget class="QWidget" name="dockWidgetContents_x">
    <layout class="QGridLayout" name="gridLayout">
     <item row="0" column="0">
      <widget class="ControlPanel" name="controlPanel" native="true">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Minimum" vsizetype="Preferred">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
  </widget>
  <action name="actionSaveData">
   <property name="text">
    <string>保存数据(S)</string>
   </property>
  </action>
  <action name="actionLoadData">
   <property name="text">
    <string>读入数据(R)</string>
   </property>
  </action>
  <action name="action_3">
   <property name="text">
    <string>退出(X)</string>
   </property>
  </action>
  <action name="actionSpiCtrl">
   <property name="text">
    <string>SPI控制(C)</string>
   </property>
  </action>
  <action name="actionFftDepth">
   <property name="text">
    <string>FFT深度(D)</string>
   </property>
  </action>
  <action name="actionSpectrum">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>频谱分析(&amp;F)</string>
   </property>
  </action>
  <action name="actionLogic">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>逻辑分析(&amp;L)</string>
   </property>
  </action>
  <action name="action_7">
   <property name="text">
    <string>帮助主题(&amp;T)</string>
   </property>
  </action>
  <action name="action_AboutAdcAnalyzer">
   <property name="text">
    <string>关于ADC Analyzer（&amp;A）</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <customwidgets>
  <customwidget>
   <class>ControlPanel</class>
   <extends>QWidget</extends>
   <header>controlpanel.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources>
  <include location="mainwindow.qrc"/>
 </resources>
 <connections/>
</ui>
